image/docker/schema2/config.py

```diff
@@ -64,11 +64,11 @@
   "history": [
     {
       "created": "2018-04-03T18:37:09.284840891Z",
-      "created_by": "\/bin\/sh -c #(nop) ADD file:9e4ca21cbd24dc05b454b6be21c7c639216ae66559b21ba24af0d665c62620dc in \/ "
+      "created_by": "/bin/sh -c #(nop) ADD file:9e4ca21cbd24dc05b454b6be21c7c639216ae66559b21ba24af0d665c62620dc in / "
     },
     {
       "created": "2018-04-03T18:37:09.613317719Z",
-      "created_by": "\/bin\/sh -c #(nop)  CMD [\"sh\"]",
+      "created_by": "/bin/sh -c #(nop)  CMD ["sh"]",
```

Comment:
`\"` seems to be legit, perhaps it should be `\\"` or the whole string should be `r"..."`

Reply:
good catch!

Reply:
fixed.